__tests__/api.test.js

```diff
@@ -92,4 +92,55 @@ describe(' GET /api/reviews/:review_id', () => {
             .get('/api/reviews/1000')
             .expect(404)
     });
+});
+
+describe(' 6. GET /api/reviews/:review_id/comments', () => {
+    test('returns correct comments', () => {
+        return request(app)
+            .get('/api/reviews/2/comments')
+            .expect(200)
+            .then((res) => {
+                expect(res.body).toEqual([
+                    {
+                      comment_id: 1,
+                      body: 'I loved this game too!',
+                      review_id: 2,
+                      author: 'bainesface',
+                      votes: 16,
+                      created_at: '2017-11-22T12:43:33.389Z'
+                    },
+                    {
+                      comment_id: 4,
+                      body: 'EPIC board game!',
+                      review_id: 2,
+                      author: 'bainesface',
+                      votes: 16,
+                      created_at: '2017-11-22T12:36:03.389Z'
+                    },
+                    {
+                      comment_id: 5,
+                      body: 'Now this is a story all about how, board games turned my life upside down',
+                      review_id: 2,
+                      author: 'mallionaire',
+                      votes: 13,
+                      created_at: '2021-01-18T10:24:05.410Z'
+                    }
+                  ])
+            })
+    });
+
+    test('with commentless review', () => {
+        return request(app)
+            .get('/api/reviews/1/comments')
+            .expect(200)
+            .then((res) => {
+                expect(res.body.msg).toBe('no comments for review')
+            })
+    });
+
+    test('404s on non existent id', () => {
+        return request(app)
+            .get('/api/reviews/10000/comments')
+            .expect(404)
+    });
 });
```

Comment:
You need another test for error 400's. That's when the id is an incorrect data type (not a number)

Think about how you can take advantage of built-in SQL error handling to achieve this